couchdb/couchdb.go

```diff
@@ -58,21 +58,34 @@ func (j JSONDoc) Rev() string {
 // DocType returns the document type of the document
 //   "io.cozy.event" == doc.Doctype()
 func (j JSONDoc) DocType() string {
-	qid, ok := j["_id"].(string)
-	if !ok {
-		return ""
-	}
-	return qid[0:strings.Index(qid, "/")]
+	return j.Type
 }
 
 // SetID is used to set the qualified identifier of the document
```

Comment:
`s/qualified indentifier/identifier/` (there may be other comments using this term)